src/test/resources/test.properties.template

```diff
@@ -12,6 +12,9 @@ s3.secret_key=<secretkey>
 ## Endpoint to the S3 Access Point
 s3.endpoint=http[s]://<ecshost>[:9020|:9021]
 
+## Uncomment this to add your sts token
+#s3.security_token=<securityToken>
+
```

Comment:
add 3 new properties for temp. credentials (see above)

Reply:
done